### PR TITLE
Create SParts.md

### PR DESCRIPTION
**Pull request by MarkGisi**
Based upon the recommendation of Brian Behlendorf and Christopher Ferris, we are submitting the following proposal for HyperLedger Labs' consideration. The overview of this project can be found here:
     https://github.com/Wind-River/sparts/blob/master/README.md 
If you have any additional questions or requests you can reach me at: Mark.Gisi@windriver.com

### CODE REVIEW

File: labs/SParts.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Lab Name
+
+SParts Project
+
+# Short Description
+
+The Software Parts (SParts) project delivers a Sawtooth-based Software Parts Ledger that add accountabiity to tracking the open source components used in the construction of a software part whether it be a single library, application, container or an entire operating system runtime.
```
Comment (trbs) [inline labs/SParts.md]: Typo at: "accountabiity"

**Comment by trbs** [comment]
Could you redo the commit with `-s` / sign-off so it passes the DCO bot please ?

**Comment by trbs** [comment]
Could you elaborate a bit more on the `Pre-existing repository` section ? 

When you say to "move components over" do you mean that the entire https://github.com/Wind-River/sparts/ repository can be moved into the labs or only specific parts of it ?

If the later them maybe the proposal needs to be updated for just that scope instead of the entire `sparts` project ?

File: labs/SParts.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Lab Name
+
+SParts Project
```
Comment (tkuhrt) [inline labs/SParts.md]: Can we change any usage of project to lab in this proposal? We want to be sure that Hyperledger labs are separate from Hyperledger projects.

File: labs/SParts.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+# Scope of Lab
+
+This project allows any individual, organization, foundation or supply chain to easily create a distributed ledger to track the open source components used to construct of a collection of software solutions. The initial focus was to track the open source components from which today's manufactured products and devices are constructed. A number of important benefits are obtained by knowing which open source components are used such as: 1) ensuring manufactures are able to identify and secure the distribution rights (licenses) for all open source components; 2) understanding the impact of open source security vulnerabilities; 3) enable identification of cryptography technologies (e.g., FIPS 140-2 certification, export licensing); and 4) enable accurate reporting on all open source parts as a requirement to obtaining functional safety certification for safety critical products (e.g., medical devices, aircraft, autonomous vehicles, elevators, …)
```
Comment (tkuhrt) [inline labs/SParts.md]: "used to construct of a collection" -> "used to construct a collection"
"manufactures" -> "manufacturers"

**Comment by tkuhrt** [comment]
Can we close this in favor of #16?

**Pull request closed without merging**